core/src/test/java/org/gbif/pipelines/core/parsers/multimedia/MultimediaParserTest.java

```diff
@@ -0,0 +1,265 @@
+package org.gbif.pipelines.core.parsers.multimedia;
+
+import org.gbif.api.vocabulary.Extension;
+import org.gbif.pipelines.core.parsers.ParsedField;
+import org.gbif.pipelines.core.utils.ExtendedRecordCustomBuilder;
+import org.gbif.pipelines.io.avro.ExtendedRecord;
+import org.gbif.pipelines.io.avro.IssueType;
+import org.gbif.pipelines.io.avro.MediaType;
+
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Tests the {@link MultimediaParser}.
+ */
+public class MultimediaParserTest {
+
+  @Test
+  public void audubonExtensionSuccesfulTest() {
+
+    Map<String, String> audubonExtension =
+      new ExtendedRecordCustomBuilder.MultimediaExtensionBuilder().accessURI("http://uri.com")
+        .references("http://references.com")
+        .title("title")
+        .description("description")
+        .caption("caption")
+        .license("license")
+        .publisher("publisher")
+        .contributor("contributor")
+        .source("source")
+        .derivedFrom("derivedFrom")
+        .audience("audience")
+        .rightsHolder("rightsHolder")
+        .creator("creator")
+        .format("image/jpeg")
+        .created("10/10/2000")
+        .build();
+
+    ExtendedRecord extendedRecord =
+      new ExtendedRecordCustomBuilder().id("123").addExtensionRecord(Extension.AUDUBON, audubonExtension).build();
```

Comment:
I think, would be better add method in ExtendedRecordCustomBuilder, like:
```
public static ExtendedRecordCustomBuilder create(){
    return new ExtendedRecordCustomBuilder();
}
```
It will be like - `ExtendedRecordCustomBuilder.create().id("123")`